src/app/theme/light-theme.ts

```diff
@@ -17,5 +17,9 @@ export const lightTheme: Theme = {
     "--mborder": "#E5E5E5",
     "--filter": "",
     "--unread": "#e7effe",
+    "--topbar": "#efefef",
```

Comment:
I don't think this is the correct color for the profile top bar.  Currently it is `#2e2e2e`